src/rgw/rgw_putobj_processor.cc

```diff
@@ -91,7 +91,7 @@ int RadosWriter::process(bufferlist&& bl, uint64_t offset)
   } else {
     op.write(offset, data);
   }
-  auto c = aio->submit(stripe_obj, stripe_raw, &op, cost);
+  auto c = aio->submit(stripe_obj, stripe_raw, &op, cost, 0);
```

Comment:
I guess I don't love having to pass bare zero here, even if a fix were cosmetic it might be nice

Reply:
okay, i'll give it a name and comment